config/initializers/kaminari_config.rb

```diff
@@ -1,6 +1,10 @@
 # frozen_string_literal: true
 Kaminari.configure do |config|
-  config.default_per_page = 50
+  if Rails.env.development? || Rails.env.staging?
```

Comment:
Can we also think of making this configurable from env config? Does this makes sense here?

Reply:
That's true - we could move this to `development.rb` and `staging.rb` which is a bit neater. But I wouldn't block the PR for that.